_faq_entries/7-wildcard.md

```diff
@@ -3,10 +3,10 @@ title:  "Will Let's Encrypt issue wildcard certificates?"
 weight: 7
 ---
 
-Let's Encrypt has [begun issuing](https://community.letsencrypt.org/t/acme-v2-and-wildcard-certificate-support-is-live/55579) wildcard certificates in March 2018. Certbot has added support for wildcard certificates as of version 0.22. Obtaining a wildcard certificate requires using the DNS authentication method, either via `--manual` or via a Certbot DNS plugin appropriate to your DNS provider.
+Let's Encrypt has [begun issuing](https://community.letsencrypt.org/t/acme-v2-and-wildcard-certificate-support-is-live/55579) wildcard certificates in March 2018. Certbot has added support for wildcard certificates as of version 0.22.0. Obtaining a wildcard certificate requires using the DNS authentication method, either via `--manual` or via a Certbot DNS plugin appropriate to your DNS provider. Differently from other Let's Encrypt certificates, a wildcard certificate cannot be obtained simply by placing special files on your web server; instead, it requires the ability to update your DNS records.
 
-Note that [depending how you install Certbot](https://community.letsencrypt.org/t/getting-wildcard-certificates-with-certbot/56285), appropriate plugins to automate the process may not yet be available on your system.
+**Certificates obtained with `--manual` cannot be renewed automatically with `certbot renew`** (unless you've provided a custom authorization script). In order to obtain wildcard certificates that can be renewed without human intervention, you'll need a compatible DNS provider API, or a script that can make appropriate DNS record changes upon demand. Even if your regular DNS provider doesn't support a compatible update mechanism, you can use a `CNAME` delegation for the `_acme-challenge` record in your DNS zone to a different provider that does. You can also point `_acme-challenge` to an [acme-dns](https://github.com/joohoi/acme-dns) instance.
```

Comment:
I'd change it to: Certificates obtained with a DNS plugin can be renewed automatically with `certbot renew`, but certificates obtained with `--manual` cannot be renewed automatically with `certbot renew`.

because to new users it might not be clear that manual and dns are contrasting plugin options